src/Login.jsx

```diff
@@ -9,17 +45,23 @@ function LoginPage(props) {
             }
         >
             <h1 className="mb-3 text-4xl font-normal">Войти: </h1>
-            <form className="flex flex-col items-center rounded-xl bg-gray-200 py-4 px-10">
+            <form
+                className="flex flex-col items-center rounded-xl bg-gray-200 py-4 px-10"
+                onSubmit={submit}
+                action="/api/login"
+                method="POST"
+            >
                 <div className="mb-5 w-fit">
                     <span className="block text-center text-xl font-light">Логин: </span>
-                    <Input />
+                    <Input type="text" name="username" />
                 </div>
                 <div className="mb-4 w-fit">
                     <span className="block text-center text-xl font-light">Пароль: </span>
-                    <Input type="password" />
+                    <Input type="password" name="password" />
                 </div>
-                <ButtonSubmit text="Войти" />
+                <ButtonSubmit text="Войти" disabled={submitted} />
             </form>
+            <ToastContainer position="bottom-center" theme="light" transition={Zoom} />
```

Comment:
you don't need to specify theme and position here, because you specify it when you're creating a toast